app/views/catalog/_show_default.html.erb

```diff
@@ -1,20 +1,20 @@
 <div class='row'>
   <div class='col-md-12'>
-    <ul class='nav nav-pills' role='tablist'>
+    <ul class='nav nav-tabs nav-fill' role='tablist'>
```

Comment:
Could you add an `aria-label`to describe these tabs? If @jvine sees this before heading out, she can suggest some text here. If not, feel free to use your discretion and we can revise the text once @ggeisler is back.

Here's an example:
https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

Reply:
I propose "Context" until we find something better.

Reply:
@camillevilla Yes, good idea! I'll make a new i18n key for it and share it between the collection & component show pages.